packages/connectivity/src/scp-cf/destination/index.ts

```diff
@@ -0,0 +1,12 @@
+export * from './destination-accessor';
```

Comment:
I like the introduction of index.ts files in subfolders, have to do that as well in my PR 👍